feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt

```diff
@@ -68,8 +68,8 @@ class SearchFragment : DaggerFragment() {
 
         sessionContentsStore.sessionContents.changed(viewLifecycleOwner) { contents ->
             searchActionCreator.search(
-                searchView?.query?.toString(),
-                sessionContentsStore.sessionContents.requireValue()
+                searchStore.searchResult.value?.query,
```

Comment:
Can you add query property for SearchStore?

```
    val query get() = searchResult.value?.query
```

Reply:
Done !! plz confrim commit https://github.com/DroidKaigi/conference-app-2019/pull/238/commits/99001bbf206b2c7be50895b55886ea0dfd6bc822